app/assets/stylesheets/redesign/nav.scss

```diff
@@ -160,7 +160,7 @@ nav.navbar-second {
   }
 
   &.adult {
-    @include navbar($background: $blue-very-dark, $foreground: $white, $background-lozenge: $blue-medium, $foreground-lozenge: $blue-dark, $hover: $blue-medium);
+    @include navbar($background: $blue-very-dark, $foreground: $white, $background-lozenge: $blue-light, $foreground-lozenge: $blue-dark, $hover: $blue-light);
```

Comment:
Have switched the naming of the blues about in the palette slightly - see colours.rb for detail. The colour here is still the same as that being used before